packages/grafana-ui/src/components/FileUpload/FileUpload.tsx

```diff
@@ -42,37 +43,48 @@ export const FileUpload: FC<Props> = ({
 
   return (
     <>
-      <label className={cx(style.button, className)}>
+      <input
+        type="file"
+        id="fileUpload"
+        className={style.fileUpload}
+        onChange={onChange}
+        multiple={false}
+        accept={accept}
+        data-testid="fileUpload"
```

Comment:
Is there any value in adding this to our e2e-selectors? 🤔

Reply:
Switched to using e2e-selectors